recipe/meta.yaml

```diff
@@ -30,83 +29,88 @@ requirements:
   run:
```

Comment:
Updated versions of all dependencies explicitly declared in `pyproject.toml` to reflect those in `environments/conda-linux-64.lock.yml` and those direct dependencies that **should** have been in `pyproject.toml` but weren't.

Obsolete dependencies removed:
- `datapackage` (replaced by `frictionless`)
- `recordlinkage` (now using `sklearn` directly)

Missing dependencies added:
- `packaging`
- `pytz`
- `regex`
- `requests`
- `sqlite`
- `universal_pathlib`
- `urllib3`


New dependencies added:
- `jellyfish`
- `frictionless`